### ISSUE TITLE
Version 4.6.9 can not work in Revit 2015&2017

### ISSUE THREAD

**Issue by WinterXMQ** [eirannejad/pyRevit#456]
I try v4.6.8 and v4.6.9, v4.6.9 works final in Revit 2018 & 2019, but fail in Revit 2015&2017.
![qq 20181231121007](https://user-images.githubusercontent.com/3694982/50554423-14fa9c80-0cf5-11e9-8be7-48eb7418287a.jpg)

The error in Revit 2015 is the same as that in Revit 2017.

```
==> Registered Clones (full git repos)
Name: "master" | Path: "D:\Program Files\pyRevit"
==> Registered Clones (deployed from archive)
==> Attachments
Autodesk Revit 2019.0.2 | Clone: "master" | Engine: "277"
Autodesk Revit 2018 First Customer Ship | Clone: "master" | Engine: "277"
Autodesk Revit 2019 First Customer Ship | Clone: "master" | Engine: "277"
==> Installed UI Extensions
==> Installed Library Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2019.0.2 | Version: 19.0.20.1 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit 2019\"
Autodesk Revit 2018 First Customer Ship | Version: 18.0.0.420 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit Content 2018 Simplified Chinese\"
Autodesk Revit 2019 First Customer Ship | Version: 19.0.0.405 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit Content 2019 Simplified Chinese\"
==> Running Revit Instances
PID: 35524 | Autodesk Revit 2019.0.2 | Version: 19.0.20.1 | Language: 0 | Path: ""
Error: 值不能为 null。
参数名: path3
Run with "--debug" option to see debug messages
```

The command `pyrevit` can not recognize Revit 2015 and 2017, here are the version code in my computer:
For Revit 2018: `20181015_0930,(18.0.3.6,2018.0.3)`
For Revit 2017:  `20181011_1545, (17.0.511.0, 2017.0.4)`
For Revit 2015: 
![image](https://user-images.githubusercontent.com/3694982/50554479-0b256900-0cf6-11e9-9cd6-41e70cd9fccb.png)
![image](https://user-images.githubusercontent.com/3694982/50554523-e8e01b00-0cf6-11e9-8db2-69af355bbca8.png)

**Comment by WinterXMQ** [comment]
CLI recognize Revit version is by checking Uninstall List(When the `DisplayName` is `Revit xxxx` and `DisplayVersion` in the dictionary)

It seems that the method has some problems.
* Firstly, the old version has different name, like my Revit 2015. 
* And there are some language package installed.

So I think it should test whether the file `revit.exe` is in the installed path or not, and the `DisplayName` can be matched by `^Revit\s.*\d{4}`.

**Comment by eirannejad** [comment]
Thanks for the build numbers. Added them to CLI and resolved the displayname matching issue. I'll publish the changes with 4.6.9

**Comment by WinterXMQ** [comment]
This problem does not solved in 4.6.9(bin 0.2.17).
It does not work in Revit 2015&2017.
![image](https://user-images.githubusercontent.com/3694982/50571564-bdd5f400-0de8-11e9-9b07-c6e4fbad41ab.png)
It seems that MahApps.Metro does not work in Revit 2015&2017.

```
==> Registered Clones (full git repos)
Name: "master" | Path: "D:\Program Files\pyRevit"
==> Registered Clones (deployed from archive)
==> Attachments
Autodesk Revit 2017.0.4 | Clone: "master" | Engine: "277"
Autodesk Revit 2018.0.3 | Clone: "master" | Engine: "277"
Autodesk Revit 2019.0.2 | Clone: "master" | Engine: "277"
Autodesk Revit 2017 First Customer Ship | Clone: "master" | Engine: "277"
Autodesk Revit 2018 First Customer Ship | Clone: "master" | Engine: "277"
Autodesk Revit 2019 First Customer Ship | Clone: "master" | Engine: "277"
==> Installed UI Extensions
==> Installed Library Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2017.0.4 | Version: 17.0.511.0 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit 2017\"
Autodesk Revit 2018.0.3 | Version: 18.0.3.6 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit 2018\"
Autodesk Revit 2019.0.2 | Version: 19.0.20.1 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit 2019\"
Autodesk Revit 2017 First Customer Ship | Version: 17.0.416.0 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit Content 2017 Simplified Chinese\"
Autodesk Revit 2018 First Customer Ship | Version: 18.0.0.420 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit Content 2018 Simplified Chinese\"
Autodesk Revit 2019 First Customer Ship | Version: 19.0.0.405 | Language: 1033 | Path: "D:\Program Files\Autodesk\Revit Content 2019 Simplified Chinese\"
==> Running Revit Instances
PID: 21968 | Autodesk Revit 2017.0.4 | Version: 17.0.511.0 | Language: 0 | Path: ""
PID: 42940 | Autodesk Revit 2017.0.4 | Version: 17.0.511.0 | Language: 0 | Path: ""
==> User Environment
Adming Access: No
Executing User: WINTERXMQ\ming
Active User: WINTERXMQ\ming
%appdata%: "C:\Users\ming\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
```

**Comment by VDMBIM** [comment]
Hi was this ever resolved, is there a version that works for Revit 2015 or a workaround by any chance? Thanks
